### ISSUE TITLE
Endpoints not exposed in distributed build of 4.0.6

### ISSUE THREAD

**Issue by aforty** [spree/spree-storefront-api-v2-js-sdk#138]
If I understand #135 correctly it should now expose a `endpoints` object in the root that contains all the endpoint classes (Account, Authentication, etc) but it's not there.

```
console.log(require('@spree/storefront-api-v2-sdk')); // @4.0.6
```

<img width="274" alt="Screen Shot 2020-04-14 at 12 27 50" src="https://user-images.githubusercontent.com/523683/79249521-5f9d1b00-7e4b-11ea-804e-26fc7e614275.png">

**Comment by tniezg** [comment]
Hi @aforty,
https://github.com/spree/spree-storefront-api-v2-js-sdk/pull/135 was merged to `master`. It will be released in `4.1.0` this week. I'll make sure to mention this change in the release notes for `4.1.0`.

**Comment by aforty** [comment]
Oh whoops, that's my bad! I saw 4.0.6 was available on npm and just sort of assumed all the latest commits were part of it without really checking.

Looking forward to 4.1.0!
